client/registration.go

```
@@ -110,14 +115,14 @@ func addReg(w http.ResponseWriter, r *http.Request) {
 	defer s.Close()
 	c := s.DB(mongo.DbName).C(mongo.CollectionName)
 
-	count, _ := c.Find(bson.M{"name": reg.Name}).Count()
-	if count != 0 {
+	if count, _ := c.Find(bson.M{"name": reg.Name}).Count(); count != 0 {
```

Do  not ignore error using `_`. Rather check for error.

---

Also, log errors.